### PR TITLE
Regenerate SDK for new added api key management swagger spec

### PR DESCRIPTION
**Pull request by frankguodongchen**
## Description
Regenerate SDK for new added api key management swagger spec

here is the PR for the new swagger spec change:
https://github.com/Azure/azure-rest-api-specs-pr/pull/208
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ x] Title of the pull request is clear and informative.
- [ x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/ApplicationInsights/ApplicationInsights.Tests/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "profiles": {
```
Comment (shahabhijeet) [inline src/SDKs/ApplicationInsights/ApplicationInsights.Tests/Properties/launchSettings.json]: @frankguodongchen any reason you need this file?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Please use generate.cmd to generate your code.
This will generate the metadata file that is needed.
Also bump up your version if you are planning to publish this SDK.

File: src/SDKs/ApplicationInsights/ApplicationInsights.Tests/Properties/launchSettings.json (hunk 1 above)
Comment (frankguodongchen) [reply src/SDKs/ApplicationInsights/ApplicationInsights.Tests/Properties/launchSettings.json]: it was only used in my local to run the test in record mode, I can remove it.

**Comment by shahabhijeet** [comment]
@azuresdkci add to whitelist

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
@frankguodongchen you are missing updates to nuget package update in csproj, not doing that will hamper your ability to publish nuget package.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**